examples/reliable_conversation/src/tasks/llm_evaluators.py

```
@@ -0,0 +1,433 @@
+
+# Import our utilities
+import sys
+from pathlib import Path
+
+sys.path.insert(0, str(Path(__file__).parent.parent))
+
+from utils.config import get_llm_class
+from utils.logging import get_rcm_logger
+
+# We'll register tasks with the app instance passed from main.py
+app = None
```

Pylint (3.3.7)

[convention] Constant name "app" doesn't conform to UPPER_CASE naming style

(C0103)